AsyncDisplayKit/Layout/ASLayoutablePrivate.h

```diff
@@ -12,8 +12,29 @@
 
 @class ASLayoutSpec;
 @class ASLayoutOptions;
+@protocol ASLayoutable;
 
 @protocol ASLayoutablePrivate <NSObject>
-- (ASLayoutSpec *)finalLayoutableWithParent:(ASLayoutSpec *)parentSpec;
+
+/**
+ *  @abstract A display node cannot implement both calculateSizeThatFits: and layoutSpecThatFits:. This
+ *  method exists to give the user a chance to wrap an ASLayoutable in an ASLayoutSpec just before it is
+ *  added to a parent ASLayoutSpec. For example, if you wanted an ASTextNode that was always inside of an
+ *  ASInsetLayoutSpec, you could subclass ASTextNode and implement finalLayoutable so that it wraps
+ *  self in an inset spec.
+ *
+ *  Note that any ASLayoutable other than self that is returned MUST set isFinalLayoutable to YES BEFORE
```

Comment:
Here and in the implementation file, add a comma after YES — it looks a little weird to have them run together :)